04-path-security-and-networking/405-ingress-controllers/readme.adoc

```diff
@@ -296,11 +296,7 @@ kube-aws-ingress-controller, which we will use:
   "Action": [
     "acm:ListCertificates",
     "acm:DescribeCertificate",
-    "autoscaling:DescribeAutoScalingGroups",
-    "autoscaling:AttachLoadBalancers",
-    "autoscaling:DetachLoadBalancers",
-    "autoscaling:DetachLoadBalancerTargetGroups",
-    "autoscaling:AttachLoadBalancerTargetGroups",
+    "autoscaling:*",
```

Comment:
Is there a specific Action that is missing from the above Actions? I'd suggest we be explicit for security reasons…

Reply:
@christopherhein There are already a bunch of `*` permissions set, so I thought it makes sense for autoscaling, too.
The problem is if we add a new feature and it needs new permissions, I have to create PRs on a couple of other repositories. I think it makes sense to have in these workshop repositories not so specific permissions and we have in our documentation all permissions documented https://github.com/zalando-incubator/kube-ingress-aws-controller/blob/master/deploy/requirements.md

Reply:
I changed the permissions and deleted my old comment, because I got the same request in my kops PR.